packages/core/src/App/Containers/RealAccountSignup/helpers/constants.js

```diff
@@ -38,8 +38,8 @@ export const getAccountTitle = (
 
     switch (short_code) {
         case 'svg':
-            return localize('Deriv account');
         case 'iom':
+            return localize('Deriv account');
```

Comment:
This is related to `iom` landing company which includes UK residents as well. We should use the `im` residency check for this. UK clients should have the previous `Deriv synthetic` title.